src/pages/index.js

```diff
@@ -36,26 +133,107 @@ function handleImageClick(name, link) {
   imagePopup.open({ name, link });
 }
 
-// New PopupWithForm instances (userinfo integration change)
+// New PopupWithForm instances (userinfo integration change step 3)
 const editProfilePopup = new PopupWithForm("#profile-edit-modal", (data) => {
-  userInfo.setUserInfo({ name: data.title, job: data.description });
-  editProfilePopup.close(); //closes popup after editing profile
+  const submitButton = editProfilePopup._form.querySelector(".modal__button");
```

Comment:
* You've already found the submit button inside `PopupWithForm`, so you can get it in `index.js`, but you should make this class field **public** by removing the `_` (underscrore)


So, your code will be much easier here


```js
  const submitButton = editProfilePopup.submitButton

```